src/types/index.ts

```diff
@@ -0,0 +1,48 @@
+import type {
+  ErratumSchemaType,
+  ZodTrackerType,
+  TrackerSchemaType,
+  ZodAffectCVSSType,
+  AffectCVSSSchemaType,
+  ZodAffectType,
+  AffectSchemaType,
+} from '@/types/zodAffect';
+
+import type {
+  ZodFlawAcknowledgmentType,
+  ZodFlawReferenceType,
+  ZodFlawCVSSType,
+  FlawCVSSSchemaType,
+  ZodFlawCommentType,
+  ZodFlawType,
+  FlawSchemaType,
+} from '@/types/zodFlaw';
+
+import type { ZodAlertType } from '@/types/zodShared';
+
+import type {
+  ZodJiraUserAssignableType,
+  ZodJiraContributorType,
+  ZodJiraIssueType,
+} from '@/types/zodJira';
+
+export type {
+  ZodAlertType,
+  ErratumSchemaType,
+  ZodTrackerType,
+  TrackerSchemaType,
+  ZodAffectCVSSType,
+  AffectCVSSSchemaType,
+  ZodAffectType,
+  AffectSchemaType,
+  ZodFlawAcknowledgmentType,
+  ZodFlawReferenceType,
+  ZodFlawCVSSType,
+  FlawCVSSSchemaType,
+  ZodFlawCommentType,
+  ZodFlawType,
+  FlawSchemaType,
+  ZodJiraUserAssignableType,
+  ZodJiraContributorType,
+  ZodJiraIssueType,
+};
```

Comment:
I would use the re-export/ export aggegation https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export#re-exporting_aggregating

```suggestion
export type {
  ErratumSchemaType,
  ZodTrackerType,
  TrackerSchemaType,
  ZodAffectCVSSType,
  AffectCVSSSchemaType,
  ZodAffectType,
  AffectSchemaType,
} from '@/types/zodAffect';

export type {
  ZodFlawAcknowledgmentType,
  ZodFlawReferenceType,
  ZodFlawCVSSType,
  FlawCVSSSchemaType,
  ZodFlawCommentType,
  ZodFlawType,
  FlawSchemaType,
} from '@/types/zodFlaw';

export type { ZodAlertType } from '@/types/zodShared';

export type {
  ZodJiraUserAssignableType,
  ZodJiraContributorType,
  ZodJiraIssueType,
} from '@/types/zodJira';
```

Reply:
ahhhh yess I love this